### PR TITLE
+10 services

### PR DESCRIPTION
**Pull request by ifamed**

### CODE REVIEW

File: b0mb3r/services/lanet.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class Lanet(Service):
+    async def run(self):
+        if self.phone_code == "380":
+            params = {
+                "nme": self.russian_name,
+                "tel": self.formatted_phone,
+                "typ": "2",
+                "ip": "",
+                "cty": "1",
+                "srv_order_info": "Заказ консультации",
```
Comment (crinny) [inline b0mb3r/services/lanet.py]: This is a manual call, not automatic. b0mb3r doesn't accept those yet.

File: b0mb3r/services/service.py [hunk 2]
```diff
@@ -20,6 +20,13 @@ def __init__(self, phone, phone_code):
             random.choice("АаБбВвГгДдЕеЁёЖжЗзИиЙйКкЛлМмНнОоПпРрСсТтУуФфХхЦцЧчШшЩщЪъЫыЬьЭэЮюЯя")
             for _ in range(5)
         )
+        self.russian_real_name = random.choice([
+            "Александра", "Алиса", "Анастасия", "Ангелина", "Анна", "Арина",
```
Comment (crinny) [inline b0mb3r/services/service.py]: Why?

File: b0mb3r/services/lanet.py (hunk 1 above)
Comment (ifamed) [reply b0mb3r/services/lanet.py]: > This is a manual call, not automatic. b0mb3r doesn't accept those yet.

In this service SMS arrives and within a minute a call arrives.

File: b0mb3r/services/service.py (hunk 2 above)
Comment (ifamed) [reply b0mb3r/services/service.py]: > Why?

Some services with registration and calls may think that this is spam and cancel orders without a call.

Comment (crinny) [reply b0mb3r/services/service.py]: > Some services with registration and calls may think that this is spam and cancel orders without a call.

We don't need services where someone needs to think if this is spam or not. Only fully automatic calls and SMS.

File: b0mb3r/services/kontrmarka.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from service import Service
+import re
+
+
+class Kontrmarka(Service):
+    async def run(self):
+        if self.phone_code == "380":
+            response = await self.get("https://widget.kontramarka.ua/uk/widget11site19/user/login")
```
Comment (crinny) [inline b0mb3r/services/kontrmarka.py]: I don't know if b0mb3r actually needs such services with CSRF tokens at the moment. If I will ever add them, I am going to make it look better. For now, I won't consider this service

Comment (ifamed) [reply b0mb3r/services/kontrmarka.py]: > I don't know if b0mb3r actually needs such services with CSRF tokens at the moment. I want to keep the code clean and easily understandable. If I will ever add them, I am going to make it look better. For now, I won't consider this service

Ok, but this robs b0mb3r of many services.

File: b0mb3r/services/service.py (hunk 2 above)
Comment (ifamed) [reply b0mb3r/services/service.py]: > > Some services with registration and calls may think that this is spam and cancel orders without a call.
> 
> We don't need services where someone needs to think if this is spam or not. Only fully automatic calls and SMS.

Ok I got it.

File: b0mb3r/services/kontrmarka.py (hunk 3 above)
Comment (crinny) [reply b0mb3r/services/kontrmarka.py]: > Ok, but this robs b0mb3r of many services.

I know. The main thing I wanna keep is simplicity.

**Comment by crinny** [comment]
Will it be OK if I close the Pull Request and add the approved services manually? @ifamed

**Comment by timoniq** [comment]
> Will it be OK if I close the Pull Request and add the approved services manually? @ifamed

yes crinya thats ok

**Comment by ifamed** [comment]
> Will it be OK if I close the Pull Request and add the approved services manually? @ifamed

Ok fine

**Pull request closed without merging**